src/shared/utils.ts

```diff
@@ -3,14 +3,18 @@ import log from 'electron-log';
 import { join } from 'path';
 import http from 'http';
 import https from 'https';
+import { existsSync } from 'fs';
 
 /**
  * setup logging to store log files in ~/.polar/logs/ dir
  */
 export const initLogger = () => {
   log.transports.file.resolvePath = (variables: log.PathVariables) => {
     const ap = app || remote.app;
-    return join(ap.getPath('home'), '.polar', 'logs', variables.fileName as string);
+    const home = ap.getPath('home');
+    const xdgPath = join(home, '.local/share/polar');
```

Comment:
I haven't tested but I believe this will fail on Windows because it uses the `\` path separator. 

Use `join(home, '.local', 'share', 'polar')` to make it compatible with all platforms.